### DISCUSSION TITLE
State of the project

### DISCUSSION THREAD

**Discussion by JoernSchimmelpfeng** [spring-attic/spring-statemachine#1091] [General]
Hi there!

Thanks for providing this cool peace of technology. But I'm wondering what the plans of the statemachine are. There is very little activity lately on the project. Is it still save to keep using it? I would love to hear a positive answer here.

Bye
Jörn

**Comment by josephnatale** [comment]
> I would love to hear a positive answer here.

+1

We are also looking for some clarity here, activity seems to have dwindled after the release of 3.0.0.

**Comment by keeslips** [comment]
See also issue [https://github.com/spring-projects/spring-statemachine/issues/1161](url)
